Makefile

```diff
@@ -160,9 +160,15 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+.PHONY: image-build-common
+image-build-common:
+	$(IMAGE_BUILDER) build --no-cache \
+		--build-arg GOLANG_VERSION=$(TOOLBOX_GOLANG_VERSION) \
+		-f Dockerfiles/$(DOCKERFILE)  $(IMAGE_BUILD_FLAGS) -t $(IMG) .
+
 .PHONY: image-build
-image-build: unit-test ## Build image with the manager.
-	$(IMAGE_BUILDER) build --no-cache -f Dockerfiles/Dockerfile  ${IMAGE_BUILD_FLAGS} -t $(IMG) .
+image-build: DOCKERFILE=Dockerfile
```

Comment:
```suggestion

```
I think we can set flag with default value i.e `DOCKERFILE ?=Dockerfile` . Pass DOCKERFILE where we need it else it will take default value.

Reply:
No, I do not want to use recursive make. What exactly you do not like?